### PR TITLE
Update changelog.md

### PR DESCRIPTION
**Pull request by patrikmiksovsky95**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## 19th Match 2019 22:00 UTC
```
Comment (patrikmiksovsky95) [inline changelog.md]:
```suggestion
## 26th March 2020 12:00 UTC
```

File: changelog.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 26th March 2020 12:00 UTC
+
+[Amount](mews-api.md#amount) extended in [Process Group](mews-api.md#process-group). So net pricing is supported in reservations also in inventory.
+
 ## 19th Match 2019 22:00 UTC
```
Comment (patrikmiksovsky95) [inline changelog.md]:
```suggestion
## 19th March 2019 22:00 UTC
```

File: changelog.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## 19th Match 2019 22:00 UTC
+## 26th March 2020 12:00 UTC
+
+[Amount](mews-api.md#amount) extended in [Process Group](mews-api.md#process-group). So net pricing is supported in reservations also in inventory.
+
```
Comment (Ksenia-Galtsova) [inline changelog.md]: Maybe rephrase it to "Net pricing is supported in processGroup and inventory"

File: changelog.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## 19th Match 2019 22:00 UTC
+## 26th March 2020 12:00 UTC
+
+[Amount](mews-api.md#amount) extended in [Process Group](mews-api.md#process-group). So net pricing is supported in reservations also in inventory.
```
Comment (allisonchihak) [inline changelog.md]:
```suggestion
[Amount](mews-api.md#amount) extended in [Process Group](mews-api.md#process-group) and [Price](channel-manager-api.md#price) extended in [Update Prices](channel-manager-api.md#update-prices). Net pricing is now supported for inventory upload and reservation delivery.
```
